src/main/java/Ui.java

```diff
@@ -0,0 +1,23 @@
+public class Ui {
+    private final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    public void greet() {
+        String greeting = "____________________________________________________________\n"
```

Comment:
would it be better to abstract out this line as a variable since you are using it often?